### ISSUE TITLE
training set and test set

### ISSUE THREAD

**Issue by WHDworld** [yyxssm/PointSSC#5]
Thank you very much for your excellent work, but I would like to ask where you have split your training set and test set.

**Comment by yyxssm** [comment]
The training and testing set information are stored in pickle files, you can find both [time split setting](https://github.com/yyxssm/PointSSC/tree/main/data/V2X-Seq-SPD/time_split) and [scene split setting](https://github.com/yyxssm/PointSSC/tree/main/data/V2X-Seq-SPD/scene_split) here.
